Error from building

Hello, Upon building the sample project, got build errors.
Attaching log here from the failed build.

===================================================================
```
Compiler message:
../../../.pub-cache/hosted/pub.dartlang.org/syncfusion_flutter_maps-18.2.56-beta/lib/src/features/maps_tooltip.dart:156:3: Error: Type 'MouseCursor' not found.
  MouseCursor get cursor => MouseCursor.defer;
  ^^^^^^^^^^^
lib/samples/gauge/animation/radial_easeanimation.dart:49:11: Error: No named parameter with the name 'needsRotateLabels'.
          needsRotateLabels: true,
          ^^^^^^^^^^^^^^^^^
../../../.pub-cache/hosted/pub.dartlang.org/syncfusion_flutter_gauges-18.2.56/lib/src/radial_gauge/axis/radial_axis.dart:25:3: Context: Found this candidate, but the arguments don't match.
  RadialAxis(
  ^^^^^^^^^^
lib/samples/gauge/animation/radial_slowmiddle.dart:36:11: Error: No named parameter with the name 'needsRotateLabels'.
          needsRotateLabels: true,
          ^^^^^^^^^^^^^^^^^
../../../.pub-cache/hosted/pub.dartlang.org/syncfusion_flutter_gauges-18.2.56/lib/src/radial_gauge/axis/radial_axis.dart:25:3: Context: Found this candidate, but the arguments don't match.
  RadialAxis(
  ^^^^^^^^^^
lib/samples/gauge/axis_feature/custom_labels.dart:40:11: Error: No named parameter with the name 'needsRotateLabels'.
          needsRotateLabels: true,
          ^^^^^^^^^^^^^^^^^
../../../.pub-cache/hosted/pub.dartlang.org/syncfusion_flutter_gauges-18.2.56/lib/src/radial_gauge/axis/radial_axis.dart:25:3: Context: Found this candidate, but the arguments don't match.
  RadialAxis(
  ^^^^^^^^^^
lib/samples/gauge/axis_feature/multiple_axis.dart:55:9: Error: No named parameter with the name 'needsRotateLabels'.
        needsRotateLabels: true,
        ^^^^^^^^^^^^^^^^^
../../../.pub-cache/hosted/pub.dartlang.org/syncfusion_flutter_gauges-18.2.56/lib/src/radial_gauge/axis/radial_axis.dart:25:3: Context: Found this candidate, but the arguments don't match.
  RadialAxis(
  ^^^^^^^^^^
lib/samples/gauge/axis_feature/tick_customization.dart:33:11: Error: No named parameter with the name 'needsRotateLabels'.
          needsRotateLabels: true,
          ^^^^^^^^^^^^^^^^^
../../../.pub-cache/hosted/pub.dartlang.org/syncfusion_flutter_gauges-18.2.56/lib/src/radial_gauge/axis/radial_axis.dart:25:3: Context: Found this candidate, but the arguments don't match.
  RadialAxis(
  ^^^^^^^^^^
lib/samples/gauge/ranges/range_thickness.dart:33:11: Error: No named parameter with the name 'needsRotateLabels'.
          needsRotateLabels: true,
          ^^^^^^^^^^^^^^^^^
../../../.pub-cache/hosted/pub.dartlang.org/syncfusion_flutter_gauges-18.2.56/lib/src/radial_gauge/axis/radial_axis.dart:25:3: Context: Found this candidate, but the arguments don't match.
  RadialAxis(
  ^^^^^^^^^^
lib/samples/gauge/showcase/gauge_compass.dart:39:13: Error: No named parameter with the name 'needsRotateLabels'.
            needsRotateLabels: true,
            ^^^^^^^^^^^^^^^^^
../../../.pub-cache/hosted/pub.dartlang.org/syncfusion_flutter_gauges-18.2.56/lib/src/radial_gauge/axis/radial_axis.dart:25:3: Context: Found this candidate, but the arguments don't match.
  RadialAxis(
  ^^^^^^^^^^

Compiler message:
../../../.pub-cache/hosted/pub.dartlang.org/syncfusion_flutter_maps-18.2.56-beta/lib/src/features/maps_tooltip.dart:156:3: Error: Type 'MouseCursor' not found.
  MouseCursor get cursor => MouseCursor.defer;
  ^^^^^^^^^^^
../../../.pub-cache/hosted/pub.dartlang.org/syncfusion_flutter_maps-18.2.56-beta/lib/src/features/maps_tooltip.dart:156:29: Error: The getter 'MouseCursor' isn't defined for the class '_RenderMapTooltip'.
 - '_RenderMapTooltip' is from 'package:syncfusion_flutter_maps/maps.dart' ('../../../.pub-cache/hosted/pub.dartlang.org/syncfusion_flutter_maps-18.2.56-beta/lib/maps.dart').
Try correcting the name to the name of an existing getter, or defining a getter or field named 'MouseCursor'.
  MouseCursor get cursor => MouseCursor.defer;
                            ^^^^^^^^^^^
lib/samples/gauge/animation/radial_easeanimation.dart:49:11: Error: No named parameter with the name 'needsRotateLabels'.
          needsRotateLabels: true,
          ^^^^^^^^^^^^^^^^^
../../../.pub-cache/hosted/pub.dartlang.org/syncfusion_flutter_gauges-18.2.56/lib/src/radial_gauge/axis/radial_axis.dart:25:3: Context: Found this candidate, but the arguments don't match.
  RadialAxis(
  ^^^^^^^^^^
lib/samples/gauge/animation/radial_slowmiddle.dart:36:11: Error: No named parameter with the name 'needsRotateLabels'.
          needsRotateLabels: true,
          ^^^^^^^^^^^^^^^^^
../../../.pub-cache/hosted/pub.dartlang.org/syncfusion_flutter_gauges-18.2.56/lib/src/radial_gauge/axis/radial_axis.dart:25:3: Context: Found this candidate, but the arguments don't match.
  RadialAxis(
  ^^^^^^^^^^
lib/samples/gauge/axis_feature/custom_labels.dart:40:11: Error: No named parameter with the name 'needsRotateLabels'.
          needsRotateLabels: true,
          ^^^^^^^^^^^^^^^^^
../../../.pub-cache/hosted/pub.dartlang.org/syncfusion_flutter_gauges-18.2.56/lib/src/radial_gauge/axis/radial_axis.dart:25:3: Context: Found this candidate, but the arguments don't match.
  RadialAxis(
  ^^^^^^^^^^
lib/samples/gauge/axis_feature/multiple_axis.dart:55:9: Error: No named parameter with the name 'needsRotateLabels'.
        needsRotateLabels: true,
        ^^^^^^^^^^^^^^^^^
../../../.pub-cache/hosted/pub.dartlang.org/syncfusion_flutter_gauges-18.2.56/lib/src/radial_gauge/axis/radial_axis.dart:25:3: Context: Found this candidate, but the arguments don't match.
  RadialAxis(
  ^^^^^^^^^^
lib/samples/gauge/axis_feature/tick_customization.dart:33:11: Error: No named parameter with the name 'needsRotateLabels'.
          needsRotateLabels: true,
          ^^^^^^^^^^^^^^^^^
../../../.pub-cache/hosted/pub.dartlang.org/syncfusion_flutter_gauges-18.2.56/lib/src/radial_gauge/axis/radial_axis.dart:25:3: Context: Found this candidate, but the arguments don't match.
  RadialAxis(
  ^^^^^^^^^^
lib/samples/gauge/ranges/range_thickness.dart:33:11: Error: No named parameter with the name 'needsRotateLabels'.
          needsRotateLabels: true,
          ^^^^^^^^^^^^^^^^^
../../../.pub-cache/hosted/pub.dartlang.org/syncfusion_flutter_gauges-18.2.56/lib/src/radial_gauge/axis/radial_axis.dart:25:3: Context: Found this candidate, but the arguments don't match.
  RadialAxis(
  ^^^^^^^^^^
lib/samples/gauge/showcase/gauge_compass.dart:39:13: Error: No named parameter with the name 'needsRotateLabels'.
            needsRotateLabels: true,
            ^^^^^^^^^^^^^^^^^
../../../.pub-cache/hosted/pub.dartlang.org/syncfusion_flutter_gauges-18.2.56/lib/src/radial_gauge/axis/radial_axis.dart:25:3: Context: Found this candidate, but the arguments don't match.
  RadialAxis(
  ^^^^^^^^^^
../../../.pub-cache/hosted/pub.dartlang.org/syncfusion_flutter_maps-18.2.56-beta/lib/src/features/maps_tooltip.dart:156:29: Error: The getter 'MouseCursor' isn't defined for the class '_RenderMapTooltip'.
 - '_RenderMapTooltip' is from 'package:syncfusion_flutter_maps/maps.dart' ('../../../.pub-cache/hosted/pub.dartlang.org/syncfusion_flutter_maps-18.2.56-beta/lib/maps.dart').
Try correcting the name to the name of an existing getter, or defining a getter or field named 'MouseCursor'.
  MouseCursor get cursor => MouseCursor.defer;
                            ^^^^^^^^^^^
Target kernel_snapshot failed: Exception: Errors during snapshot creation: null
build failed.

FAILURE: Build failed with an exception.
```

===================================================================


Hope it helps others too trying to build.

--- a/lib/samples/gauge/animation/radial_easeanimation.dart
+++ b/lib/samples/gauge/animation/radial_easeanimation.dart
@@ -46,7 +46,7 @@ SfRadialGauge getRadialEaseExample() {
               fontSize: isCardView ? 10 : 12, fontWeight: FontWeight.w500),
           labelOffset: 25,
           interval: isCardView ? 1 : _interval,
-          needsRotateLabels: true,
+          canRotateLabels: true,
           annotations: <GaugeAnnotation>[
             GaugeAnnotation(
                 positionFactor: 1,

--- a/lib/samples/gauge/animation/radial_slowmiddle.dart
+++ b/lib/samples/gauge/animation/radial_slowmiddle.dart
@@ -33,7 +33,7 @@ SfRadialGauge getRadialSlowMiddleAnimation() {
           minimum: 0,
           maximum: 12,
           interval: 1,
-          needsRotateLabels: true,
+          canRotateLabels: true,
           majorTickStyle: MajorTickStyle(
               length: 0.15, lengthUnit: GaugeSizeUnit.factor, thickness: 1.5),
           minorTicksPerInterval: 4,

--- a/lib/samples/gauge/axis_feature/custom_labels.dart
+++ b/lib/samples/gauge/axis_feature/custom_labels.dart
@@ -37,7 +37,7 @@ Widget getGaugeCustomLabels() {
           axisLineStyle: AxisLineStyle(
               thicknessUnit: GaugeSizeUnit.factor, thickness: 0.1),
           interval: 10,
-          needsRotateLabels: true,
+          canRotateLabels: true,
           axisLabelStyle: GaugeTextStyle(fontSize: 12),
           minorTicksPerInterval: 0,
           majorTickStyle: MajorTickStyle(

--- a/lib/samples/gauge/axis_feature/multiple_axis.dart
+++ b/lib/samples/gauge/axis_feature/multiple_axis.dart
@@ -52,7 +52,7 @@ class _MultipleAxisExampleState extends SampleViewState {
             ? isCardView ? 0.5 : 0.6
             : 0.5,
         labelOffset: 15,
-        needsRotateLabels: true,
+        canRotateLabels: true,
         minorTickStyle: MinorTickStyle(
             color: const Color(0xFF00A8B5),
             thickness: 1.5,

--- a/lib/samples/gauge/axis_feature/tick_customization.dart
+++ b/lib/samples/gauge/axis_feature/tick_customization.dart
@@ -30,7 +30,7 @@ SfRadialGauge getRadialTickCustomization() {
           showLastLabel: false,
           startAngle: 270,
           endAngle: 270,
-          needsRotateLabels: true,
+          canRotateLabels: true,
           labelsPosition: ElementsPosition.outside,
           axisLabelStyle: GaugeTextStyle(fontSize: 12),
           majorTickStyle: MajorTickStyle(

--- a/lib/samples/gauge/ranges/range_thickness.dart
+++ b/lib/samples/gauge/ranges/range_thickness.dart
@@ -30,7 +30,7 @@ SfRadialGauge getRangeThicknessExampleGauge() {
           ticksPosition: ElementsPosition.outside,
           labelsPosition: ElementsPosition.outside,
           radiusFactor: 0.9,
-          needsRotateLabels: true,
+          canRotateLabels: true,
           majorTickStyle: MajorTickStyle(
             length: 0.1,
             thickness: 1.5,

--- a/lib/samples/gauge/showcase/gauge_compass.dart
+++ b/lib/samples/gauge/showcase/gauge_compass.dart
@@ -36,7 +36,7 @@ class _GaugeCompassExampleState extends SampleViewState {
             showAxisLine: false,
             radiusFactor: 1,
             showLastLabel: false,
-            needsRotateLabels: true,
+            canRotateLabels: true,
             tickOffset: 0.32,
             offsetUnit: GaugeSizeUnit.factor,
             onLabelCreated: _axisLabelCreated,

--- a/pubspec.yaml
+++ b/pubspec.yaml
@@ -16,15 +16,15 @@ dependencies:
   url_launcher: ^5.0.3
   intl: ^0.16.0
   path_provider: ^1.6.1
-  syncfusion_flutter_datagrid: ^18.2.44
-  syncfusion_flutter_calendar: ^18.2.44-beta
-  syncfusion_flutter_datepicker: ^18.2.44-beta
-  syncfusion_flutter_charts: ^18.2.44
-  syncfusion_flutter_gauges: ^18.2.44
-  syncfusion_flutter_sliders: ^18.2.44-beta
-  syncfusion_flutter_pdf: ^18.2.44-beta
-  syncfusion_flutter_barcodes: ^18.2.44-beta
-  syncfusion_flutter_maps: ^18.2.44-beta
+  syncfusion_flutter_datagrid: ^18.2.56-beta
+  syncfusion_flutter_calendar: ^18.2.56
+  syncfusion_flutter_datepicker: ^18.2.56-beta
+  syncfusion_flutter_charts: ^18.2.56
+  syncfusion_flutter_gauges: ^18.2.56
+  syncfusion_flutter_sliders: ^18.2.56-beta
+  syncfusion_flutter_pdf: ^18.2.56-beta
+  syncfusion_flutter_barcodes: ^18.2.56-beta
+  syncfusion_flutter_maps: ^18.2.56-beta
   
 flutter:
   uses-material-design: true

